### PR TITLE
Sync Community Code of Conduct from CNCF upstream

### PR DESCRIPTION
**Pull request by tuesdayinsighter**
There seems a typo here, `ocurring` should be `occurring` 🤔

### CODE REVIEW

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by stmcginnis** [comment]
One more occurrence in the zh-cn docs that should be picked up after this lands.

/lgtm

**Comment by tuesdayinsighter** [comment]
> One more occurrence in the zh-cn docs that should be picked up after this lands.
> 
> /lgtm

PR created at: https://github.com/kubernetes/website/pull/46052

**Review by dipesh-rawat — COMMENTED** [review]
/hold

File: content/en/community/static/cncf-code-of-conduct.md
Comment (dipesh-rawat) [inline content/en/community/static/cncf-code-of-conduct.md:1]: At the top of this file, we have a following [note (here)](https://github.com/kubernetes/website/blob/main/content/en/community/static/cncf-code-of-conduct.md?plain=1#L1) advising against direct editing:

>   <-- Do not edit this file directly. Get the latest from
     https://github.com/cncf/foundation/blob/main/code-of-conduct.md -->

I see the original upstream [file](https://github.com/cncf/foundation/blob/main/code-of-conduct.md?plain=1#L89) has fixed this typo as well. However, there appear to be additional changes that haven't been synchronized yet. Ideally, we should sync these changes from the upstream file. 

@tuesdayinsighter, would you be willing to synchronize the contents of the differences from the original upstream [file](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) at commit level https://github.com/cncf/foundation/commit/71412bb029090d42ecbeadb39374a337bfb48a9c? Also, once that's done, please update the commit level in another file **[community/code-of-conduct.md](https://github.com/kubernetes/website/blob/main/content/en/community/code-of-conduct.md)** in this line [(click here)](https://github.com/kubernetes/website/blob/main/content/en/community/code-of-conduct.md?plain=1#L10).

**Comment by dipesh-rawat** [comment]
/lgtm cancel
See https://github.com/kubernetes/website/pull/46038#discussion_r1582237235

**Comment by divya-mohan0209** [comment]
Thanks @dipesh-rawat ! Good catch :)

File: content/en/community/static/cncf-code-of-conduct.md
Comment (tuesdayinsighter) [reply content/en/community/static/cncf-code-of-conduct.md:1]: Done, file is synced at https://github.com/kubernetes/website/pull/46038/commits/06b1ec220b765556677e984487905f5d05103b89
and community/code-of-conduct.md is synced at https://github.com/kubernetes/website/pull/46038/commits/b487589842736c9f126ad8b44809037a03ad08cc

**Comment by dipesh-rawat** [comment]
Based on https://github.com/kubernetes/website/pull/46038#discussion_r1582492760

/unhold

**Comment by dipesh-rawat** [comment]
/retitle Sync Community Code of Conduct from CNCF upstream

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix!
/lgtm

[Preview Page](https://deploy-preview-46038--kubernetes-io-main-staging.netlify.app/community/code-of-conduct/)

**Review by divya-mohan0209 — COMMENTED** [review]
Thank you so much for your contributions!
/approve

**Pull request merged**